documentation/src/main/markdown/packagemgrs/golang.md

```diff
@@ -9,14 +9,15 @@
 [detect_product_short] has four detectors for GoLang:
 
 * Go Mod CLI (GO_MOD) detector (recommended)
+* Go Mod File (GO_MOD) detector
```

Comment:
Should this be using something other than (GO_MOD) to differentiate from Go Mod CLI?

Reply:
This new detector adds buildless capability to the existing Go Modules package manager support. Therefore, it is categorized under the same entity. Detect first attempts the Go Mod CLI detector, and if it fails, it falls back to the Go Mod File buildless detector.

Reply:
Thank you.